build.zig

```diff
@@ -10,7 +10,7 @@ const zls_version_is_tagged: bool = false;
 /// ComptimeStringMap: return a regular struct and optimize
 ///
 /// Must match the `minimum_zig_version` in `build.zig.zon`.
-const minimum_zig_version = "0.13.0-dev.33+8af59d1f9";
+const minimum_zig_version = "0.13.0-dev.46+3648d7df1";
```

Comment:
Minimum Zig version should be the commit that causes errors on deprecated function usage. Additionally, this version should also be specified in `build.zig.zon`.

Reply:
Thanks for your help. This is my 2nd day learning zig. Could you help me what does the number after the `dev.` mean? The last part is the commit hash but I don't know what is the 46 or 33 was earlier

Reply:
I found the `zig version` command print it